designer/server/src/main/scala/pl/touk/nussknacker/ui/process/migrate/RemoteEnvironment.scala

```diff
@@ -30,11 +31,13 @@ import scala.concurrent.{ExecutionContext, Future}
 
 trait RemoteEnvironment {
 
+  val passUsernameInMigration: Boolean = true
+
   def compare(localProcess: DisplayableProcess, remoteProcessVersion: Option[VersionId])(implicit ec: ExecutionContext) : Future[Either[EspError, Map[String, Difference]]]
 
   def processVersions(processName: ProcessName)(implicit ec: ExecutionContext) : Future[List[ProcessVersion]]
 
-  def migrate(localProcess: DisplayableProcess, category: String)(implicit ec: ExecutionContext, loggedUser: LoggedUser) : Future[Either[EspError, Unit]]
+  def migrate(localProcess: DisplayableProcess, category: String, passUsername: Boolean)(implicit ec: ExecutionContext, loggedUser: LoggedUser) : Future[Either[EspError, Unit]]
```

Comment:
Why do you have this in a parameter and as a field? (`passUsernameInMigration`)?

Reply:
Without any reason, I've added one and forgot about a second one :facepalm: Removed.